README.md

```diff
@@ -34,73 +34,66 @@ from aicsimageio import AICSImage, imread
-img.shape  # returns tuple of dimension sizes in STCZYX order
-img.size("STC")  # returns tuple of dimensions sizes for just STC
-img.get_image_data("CZYX", S=0, T=0)  # returns 4D CZYX numpy array
-img.get_image_dask_data("CZYX", S=0, T=0)  # returns 4D CZYX dask array
+img.dask_data  # returns 5D TCZYX dask array
+img.dims.order  # returns string "TCZYX"
+img.dims.X  # returns size of X dimension
+img.shape  # returns tuple of dimension sizes in TCZYX order
+img.get_image_dask_data("CZYX", T=0)  # returns 4D CZYX dask array
+
+# Change scene
+img.set_scene(1)
+
+# Same operations on a different scene
+img.dask_data  # returns 5D TCZYX dask array
+img.dims.order  # returns string "TCZYX"
+img.dims.X  # returns size of X dimension
+img.shape  # returns tuple of dimension sizes in TCZYX order
+img.get_image_dask_data("CZYX", T=0)  # returns 4D CZYX dask array
 
 # Read specified portion of dask array
-lazy_s0t0 = img.get_image_dask_data("CZYX", S=0, T=0)  # returns 4D CZYX dask array
-s0t0 = lazy_s0t0.compute()  # returns 4D CZYX numpy array
+lazy_t0 = img.get_image_dask_data("CZYX", T=0)  # returns 4D CZYX dask array
+t0 = lazy_t0.compute()  # returns 4D CZYX numpy array
 
-# Or use normal numpy array slicing
+# Get a 5D TCZYX dask array
 lazy_data = imread_dask("my_file.tiff")
-lazy_s0t0 = lazy_data[0, 0, :]
-s0t0 = lazy_s0t0.compute()
+lazy_t0 = lazy_data[0, :]
+t0 = lazy_t0.compute()
 ```
 
-
-### Speed up IO and Processing with Dask Clients and Clusters
-If you have already spun up a `distributed.Client` object in your Python process or
-your processing is running on a distributed worker, great, you will naturally gain IO
-and processing gains. If you haven't done that or don't know what either of those are,
-there are some utility functions to help construct and manage these for you.
-
-```python
-from aicsimageio import AICSImage, dask_utils
-
-# Spawn a local cluster
-# These objects will be connected and useable for the lifespan of the context manager.
-with dask_utils.cluster_and_client() as (cluster, client):
-
-    img1 = AICSImage("1.tiff")
-    img2 = AICSImage("2.tiff")
-    img3 = AICSImage("3.tiff")
-
-    # Do your image processing work
-
-# Connect to a remote cluster
-# If you pass an address in, it will create and shutdown the client and no cluster will
-# be created. These objects will be connected and useable for the lifespan of the
-# context manager.
-with dask_utils.cluster_and_client(address="tcp://localhost:1234") as (cluster, client):
-
-    img1 = AICSImage("1.tiff")
-    img2 = AICSImage("2.tiff")
-    img3 = AICSImage("3.tiff")
-
-    # Do your image processing work
-```
-
-**Note:** The `dask_utils` module require that the processing machine or container have
-networking capabilities enabled to function properly.
-
+#### Quick Start Notes
+In short, if the word "dask" appears in the function or property name, the function
+utilizes delayed reading, if not, the underlying operation is backed by the image fully
```

Comment:
I think it would be better to word this by saying the image is likely to be entirely loaded into memory, but leave room for the underlying implementation to be smarter. Something like "...utilizes delayed reading.  If not, the requested image will be loaded immediately and the internal implementation may result in loading the entire image even if only a small chunk was requested.  Currently, `AICSImage.data` and `AICSImage.get_image_data` load and cache the entire image in memory before performing their operation.  `AICSImage.dask_data` and `AICSImage.get_image_dask_data` do not load any image data until the user calls `compute` on the `dask.Array` object and only the requested chunk will be loaded into memory instead of the entire image."

Reply:
These changes should be made on `master` and merged in to this branch. But agree!

Reply:
Resolved by #149